python/mlc_llm/interface/gen_config.py

```diff
@@ -129,7 +129,7 @@ def gen_config(  # pylint: disable=too-many-locals,too-many-arguments,too-many-b
         prefill_chunk_size=model_config.prefill_chunk_size,
         attention_sink_size=getattr(model_config, "attention_sink_size", -1),
         tensor_parallel_shards=model_config.tensor_parallel_shards,
-        conv_template=conversation,
+        conv_template=conversation,  # type: ignore
```

Comment:
Just curious why we disable mypy for this line?

Reply:
PyLance was raising a `reportArgumentType` issue for this line, attached the screenshot below:

![image](https://github.com/mlc-ai/mlc-llm/assets/68523530/472a6df7-8320-4eb4-893a-8d10e4a10c43)